src/Sankey/Sankey.tsx

```diff
@@ -64,6 +65,19 @@ export interface SankeyProps extends ChartProps {
    */
   nodePadding?: number;
 
+  /**
+   * Label position.
+   */
+  labelPosition?: SankeyLabelPosition;
+
+  /**
+   * Percentage of total width occupied by labels on 
+   * either side of the graph inside the container.
+   * Should be between (0, 1), excluding extreme values.
+   * Useful only when labels are outside the chart - `labelPosition = 'outside'`
+   */
+  labelPaddingPercent?: number;
```

Comment:
This property doesn't really make sense to me. I'd rather try to calculate this and not expose it in the api.

Reply:
The use case for this is when a user doesn't want to ellipsize long labels but also doesn't want it to flow outside the container, so the only place they can compromise is by deciding much area of the container do they want to give to the labels vs the actual chart.

Reply:
If we try to calculate it ourself, there may be a scenario where the labels are too long and the user explicitly doesn't want to ellipsize them so we end up in either of the two scenarios:-
1. The actual chart in between becomes so narrow that it's hardly visible
2. The labels overflow into the chart area though the position specified was 'outside'
Therefore, I feel it's better to allow the user to control this itself

Reply:
@shaswat-indian - I feel strongly this shouldn't be exposed so let's lets figure out how to find a good middle ground here.

> The use case for this is when a user doesn't want to ellipsize long labels but also doesn't want it to flow outside the container, so the only place they can compromise is by deciding much area of the container do they want to give to the labels vs the actual chart.

Let's let the user pass the max ellipsis length then they can decide that. I feel like chart should prioritize chart over labels.